app/actions/manager/version.ts

```diff
@@ -0,0 +1,12 @@
+import type { PayloadAction } from '../types';
+import { ActionType } from '../types';
+import type { VersionResponse } from '../../../backend/handler/AuthHandler.types';
+
+export type SetVersionAction = PayloadAction<VersionResponse, ActionType.SET_MANAGER_VERSION>;
```

Comment:
As in other components we may see such a convention that besides `Set<someName>Action` we also have exported `<fileName>Action`, what do you think about sticking to this convention? 🤔 

It would resolved in having
```suggestion
export type SetVersionAction = PayloadAction<VersionResponse, ActionType.SET_MANAGER_VERSION>;
export type VersionAction = SetVersionAction ;
```

as we don't have any other action typings in that file, but IMO it would increase the consistency and predictibility of the code 💪 

Currently we can see in `app/reducers/managerReducer/managerReducer.ts` that its usage may look a little bit odd 😁:
```typescript
type ManagerAction =
    | AuthAction
    | ClusterStatusAction
    | SetVersionAction
    | TenantAction
    | LicenseAction
    | MaintenanceAction;
```

Reply:
I noticed that during implementation, but was not sure if I should change it.

But if you see the same, I have applied the change 👍🏻